doc/man/8/crushtool.rst

```diff
@@ -134,10 +136,10 @@ documented by the **--help-output** option.
 
 .. option:: --output-csv
 
-   create CSV files (in the current directory) containing information
+   Creates CSV files (in the current directory) containing information
    documented by **--help-output**. The files are named after the rule
    used when collecting the statistics. For instance, if the rule
-   metadata is used, the CSV files will be::
+   : 'metadata' is used, the CSV files will be::
```

Comment:
Yes, I was trying to give focus on the rule named metadata which earlier was but ambigous. This way, user can indentify more clearly that metadata is the rule we are talking about.